test/test_property.py

```diff
@@ -70,52 +65,52 @@ def test_simple_attributes(self):
         self.assertIsNone(prop.dependency_value)
 
     def test_value(self):
-        p = Property("property", 100)
-        self.assertEqual(p.values[0], 100)
-        self.assertIsInstance(p.values, list)
+        prop = Property("property", 100)
+        self.assertEqual(prop.values[0], 100)
+        self.assertIsInstance(prop.values, list)
 
-        p.values = None
-        self.assertEqual(len(p), 0)
+        prop.values = None
+        self.assertEqual(len(prop), 0)
 
-        p.values = [1, 2, 3]
-        p.values = ""
-        self.assertEqual(len(p), 0)
+        prop.values = [1, 2, 3]
+        prop.values = ""
+        self.assertEqual(len(prop), 0)
 
-        p.values = [1, 2, 3]
-        p.values = []
-        self.assertEqual(len(p), 0)
+        prop.values = [1, 2, 3]
+        prop.values = []
+        self.assertEqual(len(prop), 0)
 
-        p.values = [1, 2, 3]
-        p.values = ()
-        self.assertEqual(len(p), 0)
+        prop.values = [1, 2, 3]
+        prop.values = ()
+        self.assertEqual(len(prop), 0)
 
-        p.values.append(5)
-        self.assertEqual(len(p.values), 0)
+        prop.values.append(5)
+        self.assertEqual(len(prop.values), 0)
 
-        p2 = Property("test", {"name": "Marie", "name": "Johanna"})
-        self.assertEqual(len(p2), 1)
+        prop2 = Property("test", {"name": "Marie"})
```

Comment:
oh, no, you destroyed the old pun: "Marie Johanna" ;)

Reply:
I did not even get that one... makes me sad, but I shall be strong and endure it. :p